### ISSUE TITLE
SymlinkAndCleanup - make cleanup asynchronous and 

### ISSUE THREAD

**Issue by ctheune** [flyingcircusio/batou_ext#163]
At least one customer has been bitten by the fact that cleanups may take a *very* long time. 

The cleanup IMHO should spawn a systemd-run job for the `rm -rf` and limit IOPS to something like 100 iops/s to ensure the cleanup job doesn't overload the system.

**Comment by EdwardBock** [comment]
I'm back. Any news?

**Comment by elikoga** [comment]
Yes, the `use_systemd_run_async_cleanup` option on SymlinkAndCleanup added in #159 has this behaviour

**Comment by EdwardBock** [comment]
We tested this. This was not working. @zagy knows more about the problems as he was talking to my colleague.
